Assets/Plugins/iOS/Shopify/BuyTests/PaymentSessionTests.swift

```diff
@@ -68,7 +68,7 @@ class PaymentSessionTests: XCTestCase {
     //  MARK: - Supported Networks -
     //
     func testSupportedNetworks() {
-        XCTAssertEqual(PaymentSession.supportedNetworks, [.amex, .masterCard, .visa, .discover])
```

Comment:
I'd propose we take it out for now, the PRs are quicker to review when its just one concise change per PR.